### DISCUSSION TITLE
Getting Started with XIAO nRF52840 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#965] [Q&A]
# Getting Started with XIAO nRF52840 | Seeed Studio Wiki

Getting Started with Seeed Studio XIAO nRF52840 (Sense)

https://wiki.seeedstudio.com/XIAO_BLE/

**Comment by Harshonar** [comment]
Hello,
I am using Xiao-BLE sense board with ability to drive it with a Lithium Ion battery (LIR2032) that has lower cutoff voltage of around 2.75V.

My question is how can I use the battery so that it cuts of driving the current at 2.5V or below? Is there any protection circuit inside that will stop the supply voltage lines some how from drawing current to safe guard the battery?

**Reply by a-usr** [reply]
See nRF52840 Documentation:
- Page 68, Item `5.3.1.6 Power-fail comparator`

and
- Page 73, Item `5.3.6.6 Brownout reset`

and associated Items.

[View nRF52840 Documentation Hosted by SEEED Studio](https://files.seeedstudio.com/wiki/XIAO-BLE/nRF52840_PS_v1.5.pdf)
[Download nRF52840 Documentation from Nordic Semiconductors](https://docs.nordicsemi.com/bundle/nRF52840_PS_v1.5/resource/nRF52840_PS_v1.5.pdf) (Same version (v1.5) as hosted by SEEED Studio)
[Download nRF52840 Documentation from Nordic Semiconductors](https://docs-be.nordicsemi.com/bundle/ps_nrf52840/attach/nRF52840_PS_v1.10.pdf) (Latest version (v1.10), possibly Inaccurate?)

[View Interactive Documentation Hosted by Nordic Semiconductors](https://docs.nordicsemi.com/bundle/ps_nrf52840/page/keyfeatures_html5.html) (Latest version (v1.10), may also be inaccurate)

**Comment by faizannazir** [comment]
how to flash bootloader in it

**Reply by a-usr** [reply]
You need a Debug Probe

**Comment by LumingLin** [comment]
Every time after I uploaded the code, the device disappeared from COM Port. If this happens to you, just click the reset button twice to enter bootloader mode.

**Reply by whooptrooper** [reply]
Double tap the reset button and it will re-appear. I had the same issue after doing the blink code test.

**Comment by aerialist** [comment]
Hi,
Thanks for providing KiCad file! It gives me a jump start.
But this v1.1 file has problem that Reset/GND and SWDIO/SWCLK are labeled on wrong pin. I assembled and noticed I cannot connect to nRF54 at all. It turned out that Reset pin is connected to GND pad... 
https://files.seeedstudio.com/wiki/XIAO-BLE/SeeedStudio_XIAO_nRF52840_v1.1_SCH&PCB.zip

**Comment by jbcuningham** [comment]
I don't see the board manager in Arduino.
